lib/airbrake-ruby/async_sender.rb

```diff
@@ -24,6 +24,8 @@ def initialize(config)
     #   library
     # @return [nil]
     def send(notice)
+      return will_not_deliver(notice) if @unsent.size >= @unsent.max
```

Comment:
Are you sure about this condition: `@unsent.size >= @unsent.max`? Shouldn't it be `@unsent.size == @unsent.max` instead? It looks like `@unsent.size` can never be greater than its capacity.